src/components/dashboard/FeedItems/ListEventItem.js

```diff
@@ -60,38 +55,63 @@ const ListEvent = ({ item: feed, theme, styles }: FeedEventProps) => {
 
 const getStylesFromProps = ({ theme }) => ({
   innerRow: {
-    padding: normalize(4),
+    alignItems: 'center',
     flexDirection: 'row',
     justifyContent: 'center',
-    alignItems: 'center',
+    padding: normalize(8),
     width: '100%',
   },
-  avatatBottom: {
+  avatarBottom: {
     alignSelf: 'flex-end',
   },
   mainSection: {
-    marginLeft: normalize(4),
+    marginLeft: normalize(8),
   },
   borderRow: {
+    alignItems: 'center',
     borderBottomStyle: 'solid',
     borderBottomWidth: normalize(2),
-    paddingBottom: normalize(4),
-    marginBottom: normalize(4),
+    display: 'flex',
+    justifyContent: 'space-between',
+    marginBottom: normalize(8),
+    paddingBottom: normalize(5),
   },
   date: {
-    fontSize: normalize(10),
-    color: 'rgba(75, 75, 75, 0.8)',
-    marginLeft: 'auto',
+    color: theme.colors.lighterGray,
     fontFamily: theme.fonts.regular,
+    fontSize: normalize(10),
+    marginTop: normalize(2),
+  },
+  bigNumberStyles: {
+    fontSize: normalize(15),
+    fontFamily: theme.fonts.bold,
   },
-  goodDollarAmount: {
+  bigNumberUnitStyles: {
     fontSize: normalize(10),
     fontFamily: theme.fonts.bold,
   },
-  rowDataSubText: {
+  bottomInfo: {
+    alignItems: 'flex-start',
+    flexShrink: 1,
+  },
+  mainInfo: {
+    alignItems: 'flex-start',
+    display: 'flex',
+    flexDirection: 'column',
+    flexGrow: 1,
+    flexShrink: 1,
+    justifyContent: 'flex-end',
+    marginTop: 'auto',
+  },
+  feedItem: {
+    marginTop: 'auto',
+  },
+  message: {
+    color: theme.colors.lighterGray,
     fontSize: normalize(10),
-    color: '#A3A3A3',
-    marginTop: normalize(4),
+    marginTop: normalize(5),
+    paddingRight: normalize(5),
```

Comment:
Same thing about values being multiple of `4`

Reply:
Changed `5` to `4`.